### ISSUE TITLE
Landrush Daemon won't start

### ISSUE THREAD

**Issue by claussni** [vagrant-landrush/landrush#109]
Landrush refuses to start the Daemon process:

```
$ vagrant landrush status
[landrush] Daemon status: crashed
Dumping daemon crash log:
=== Daemon Exception Backtrace @ 2015-05-05 14:05:13 +0200 ===
Celluloid::Task::TerminatedError: task was terminated
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/tasks/task_fiber.rb:34:in `terminate'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:345:in `each'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:345:in `cleanup'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:329:in `shutdown'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:321:in `handle_crash'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:166:in `rescue in run'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:148:in `run'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor.rb:130:in `block in start'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/thread_handle.rb:13:in `block in initialize'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/actor_system.rb:32:in `block in get_thread'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/internal_pool.rb:130:in `call'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/internal_pool.rb:130:in `block in create'
(celluloid):0:in `remote procedure call'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/calls.rb:92:in `value'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/proxies/sync_proxy.rb:33:in `method_missing'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/proxies/cell_proxy.rb:17:in `_send_'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid.rb:169:in `new'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid/supervisor.rb:10:in `supervise'
/opt/vagrant/embedded/gems/gems/celluloid-0.16.0/lib/celluloid.rb:188:in `supervise'
/home/claussni/.vagrant.d/gems/gems/rubydns-0.9.4/lib/rubydns.rb:34:in `run_server'
/home/claussni/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/server.rb:52:in `run'
/home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:92:in `block in start'
/home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:75:in `fork'
/home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:75:in `start'
/home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:37:in `daemonize'
/home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/base.rb:124:in `daemonize'
/home/claussni/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/command.rb:14:in `execute'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:42:in `execute'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:301:in `cli'
/opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.2/bin/vagrant:174:in `<main>'
=== Daemon Crashed ===
=== Daemon Stopping @ 2015-05-05 14:05:13 +0200 ===
```

Component versions:

- Vagrant 1.7.2
- landrush (0.18.0)
- ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
- nokogiri (1.6.6.2)
- Linux 3.13.0-24-generic

**Comment by ahpook** [comment]
@claussni there's not really enough in the backtrace to tell what happened. can you make a gist of a clean logfile? 
```
rm ~/.vagrant.d/data/landrush/log/Landrush-Server.log
vagrant landrush start
```
then put the new Landrush-Server.log into gist.github.com?  Thanks

**Comment by claussni** [comment]
Well, I tried this and in the end Landrush was starting fine. Here is what happened from the beginning:

```sh
[~] 
$ vagrant plugin list
vagrant-cachier (1.2.0)
vagrant-hostmanager (1.5.0)
vagrant-share (1.1.3, system)
vagrant-vbguest (0.10.0)

[~] 
$ vagrant plugin install landrush
Installing the 'landrush' plugin. This can take a few minutes...
Installed the plugin 'landrush (0.18.0)'!

[~] 
$ less ~/.vagrant.d/data/landrush/log/Landrush-Server.log 
/home/claussni/.vagrant.d/data/landrush/log/Landrush-Server.log: No such file or directory

[~] 
$ ls ~/.vagrant.d/data/landrush/log/Landrush-Server.log 
ls: cannot access /home/claussni/.vagrant.d/data/landrush/log/Landrush-Server.log: No such file or directory

[~] 
$ vagrant landrush start
[landrush] Starting daemon...
Daemon in unknown state! Will clear previous state and continue.
/home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/base.rb:106:in `initialize': No such file or directory - /home/claussni/.vagrant.d/data/landrush/log/Landrush-Server.log (Errno::ENOENT)
	from /home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/base.rb:106:in `open'
	from /home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/base.rb:106:in `crashed?'
	from /home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:134:in `status'
	from /home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:68:in `start'
	from /home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:37:in `daemonize'
	from /home/claussni/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/base.rb:124:in `daemonize'
	from /home/claussni/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/command.rb:14:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:42:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:301:in `cli'
	from /opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.2/bin/vagrant:174:in `<main>'
```

There is no Landrush-Server.log afterwards. However,  providing a file does the job:

```sh
[~] 
$ touch ~/.vagrant.d/data/landrush/log/Landrush-Server.log 

[~] 
$ vagrant landrush start
[landrush] Starting daemon...
Daemon in unknown state! Will clear previous state and continue.
[landrush] Daemon status: unknown
[landrush] Waiting for daemon to start...
[landrush] Daemon status: running pid=32283
```

Don't know if there is an actual problem which is covered by this ticket.

**Comment by BrianGilbert** [comment]
I'm having what appears to be the same problem under 10.11 (15A235d)

The guest cannot ping or lookup anything.
```
Brians-MacBook:boa-vagrant brian$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'chef/debian-7.8'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'chef/debian-7.8' is up to date...
==> default: Setting the name of the VM: boa-vagrant_default_1439182924304_66651
==> default: Fixed port collision for 22 => 2222. Now on port 2200.
==> default: Clearing any previously set network interfaces...
==> default: [landrush] Need to configure the host.
==> default: [landrush] Momentarily using sudo to put the host config in place...
==> default: [landrush] starting dns server
[landrush] Starting daemon...
Daemon in unknown state! Will clear previous state and continue.
[landrush] Daemon status: crashed
Dumping daemon crash log:
=== Daemon Exception Backtrace @ 2015-08-10 14:58:53 +1000 ===
Errno::EACCES: Permission denied - bind(2)
/Users/brian/.vagrant.d/gems/gems/rubydns-0.9.4/lib/rubydns/handler.rb:130:in `initialize'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:26:in `public_send'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:26:in `dispatch'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:79:in `dispatch'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/cell.rb:60:in `block in invoke'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/cell.rb:71:in `block in task'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/actor.rb:357:in `block in task'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/tasks.rb:57:in `block in initialize'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/tasks/task_fiber.rb:14:in `block in create'
(celluloid):0:in `remote procedure call'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:108:in `value'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/proxies/sync_proxy.rb:33:in `method_missing'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/proxies/cell_proxy.rb:17:in `_send_'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid.rb:173:in `new'
/Users/brian/.vagrant.d/gems/gems/rubydns-0.9.4/lib/rubydns/server.rb:166:in `block in run'
/Users/brian/.vagrant.d/gems/gems/rubydns-0.9.4/lib/rubydns/server.rb:147:in `each'
/Users/brian/.vagrant.d/gems/gems/rubydns-0.9.4/lib/rubydns/server.rb:147:in `run'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:26:in `public_send'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:26:in `dispatch'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:79:in `dispatch'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/cell.rb:60:in `block in invoke'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/cell.rb:71:in `block in task'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/actor.rb:357:in `block in task'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/tasks.rb:57:in `block in initialize'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/tasks/task_fiber.rb:14:in `block in create'
(celluloid):0:in `remote procedure call'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/calls.rb:108:in `value'
/Users/brian/.vagrant.d/gems/gems/celluloid-0.16.1/lib/celluloid/proxies/sync_proxy.rb:33:in `method_missing'
/Users/brian/.vagrant.d/gems/gems/rubydns-0.9.4/lib/rubydns.rb:36:in `run_server'
/Users/brian/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/server.rb:52:in `run'
/Users/brian/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:92:in `block in start'
/Users/brian/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:75:in `fork'
/Users/brian/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/controller.rb:75:in `start'
/Users/brian/.vagrant.d/gems/gems/rexec-1.6.3/lib/rexec/daemon/base.rb:129:in `start'
/Users/brian/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/action/setup.rb:49:in `start_server'
/Users/brian/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/action/setup.rb:23:in `pre_boot_setup'
/Users/brian/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/action/setup.rb:8:in `block in call'
/Users/brian/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/action/common.rb:22:in `handle_action_stack'
/Users/brian/.vagrant.d/gems/gems/landrush-0.18.0/lib/landrush/action/setup.rb:7:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/clear_network_interfaces.rb:26:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/prepare_nfs_settings.rb:18:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/synced_folders.rb:86:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/Users/brian/.vagrant.d/gems/gems/vagrant-bindfs-0.4.2/lib/vagrant-bindfs/bind.rb:14:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/synced_folder_cleanup.rb:28:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/synced_folders/nfs/action_cleanup.rb:25:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/prepare_nfs_valid_ids.rb:12:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/handle_forwarded_port_collisions.rb:160:in `handle'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/handle_forwarded_port_collisions.rb:42:in `block in call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/environment.rb:516:in `lock'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/handle_forwarded_port_collisions.rb:41:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/prepare_forwarded_port_collision_params.rb:30:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/env_set.rb:19:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/Users/brian/.vagrant.d/gems/gems/vagrant-cachier-1.2.1/lib/vagrant-cachier/action/install_buckets.rb:14:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/provision.rb:80:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/Users/brian/.vagrant.d/gems/gems/vagrant-bindfs-0.4.2/lib/vagrant-bindfs/bind.rb:14:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/Users/brian/.vagrant.d/gems/gems/vagrant-cachier-1.2.1/lib/vagrant-cachier/action/configure_bucket_root.rb:20:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/clear_forwarded_ports.rb:15:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/set_name.rb:50:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/clean_machine_folder.rb:17:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/check_accessible.rb:18:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/box_check_outdated.rb:68:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/config_validate.rb:25:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/check_virtualbox.rb:17:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/match_mac_address.rb:16:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/import.rb:32:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/customize.rb:40:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/check_accessible.rb:18:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/config_validate.rb:25:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/handle_box.rb:56:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/plugins/providers/virtualbox/action/check_virtualbox.rb:17:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/machine.rb:214:in `action_raw'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/machine.rb:191:in `block in action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/environment.rb:516:in `lock'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/machine.rb:178:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/machine.rb:178:in `action'
/opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
=== Daemon Crashed ===
=== Daemon Stopping @ 2015-08-10 14:58:53 +1000 ===

[landrush] Waiting for daemon to start...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2200 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2200
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: [landrush] setting up machine's DNS to point to our server
==> default: [landrush] network: :private_network, {:ip=>"172.16.0.50", :netmask=>"255.255.0.0", :protocol=>"tcp", :id=>"5ce3494f-384c-4113-a753-747010cb16c5"}
==> default: [landrush] network: :forwarded_port, {:guest=>22, :host=>2200, :host_ip=>"127.0.0.1", :id=>"ssh", :auto_correct=>true, :protocol=>"tcp"}
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 4.3.20
    default: VirtualBox Version: 5.0
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: [landrush] adding machine entry: boa => 172.16.0.50
==> default: Mounting shared folders...
    default: /vagrant => /Users/brian/Documents/Vagrant/boa-vagrant
    default: /tmp/vagrant-cache => /Users/brian/.vagrant.d/cache/chef/debian-7.8
==> default: Configuring cache buckets...
==> default: Running provisioner: file...
==> default: Configuring cache buckets...
==> default: Running provisioner: file...
==> default: Configuring cache buckets...
==> default: Running provisioner: shell...
    default: Running: /var/folders/q6/s6x69rqs6fbcq6w0d3hwsl040000gn/T/vagrant-shell20150810-3299-1j0mwfc.sh
==> default: stdin: is not a tty
==> default: Downloading BOA Meta installer...
==> default: Enter proxy password for user 'iCab':
==> default: curl: (6) Couldn't resolve host 'files.aegir.cc'
==> default: Running BOA Meta installer...
==> default: bash: BOA.sh.txt: No such file or directory
==> default: Installing BOA...
==> default: /tmp/vagrant-shell: line 29: boa: command not found
==> default: Configuring cache buckets...
```

**Comment by sdellow** [comment]
Also having the same problem as @BrianGilbert

**Comment by mheiges** [comment]
@BrianGilbert I think your "EACCES: Permission denied - bind "  error is due to a failed attempt to bind to privileged port 53 and is different from this original issue.  See #120

**Comment by BrianGilbert** [comment]
@mheiges I tried what was suggested in #120 and I'm still having issues

**Comment by ioquatix** [comment]
My apologies, but this is an issue with RubyDNS. Please follow ioquatix/rubydns#55 for further details.

There is a release 1.0.2 which ties RubyDNS to celluloid 0.16.0 which is an interim fix. There will be a release of RubyDNS 2.0 soon which has a better fix and also updates to celluloid 0.17.0 but that still requires a bit of work.

**Comment by hferentschik** [comment]
Closing this issue. Two of the comments are actually referring to another issue - #120, whereas the original issue seems out of date.
